packages/admin/admin/src/dataGrid/gridColumnTypes.tsx

```diff
@@ -0,0 +1,19 @@
+import { GridColTypeDef } from "@mui/x-data-grid";
+import { IntlShape } from "react-intl";
+
+const date = (intl: IntlShape): GridColTypeDef => ({
+    type: "date",
+    valueGetter: ({ value }) => value && new Date(value),
+    valueFormatter: ({ value }) => value && intl.formatDate(value, { dateStyle: "medium" }),
+});
+
+const dateTime = (intl: IntlShape): GridColTypeDef => ({
+    type: "dateTime",
+    valueGetter: ({ value }) => value && new Date(value),
+    valueFormatter: ({ value }) => value && intl.formatDate(value, { dateStyle: "medium", timeStyle: "short" }),
+});
+
+export const gridColumnTypes = {
+    date,
+    dateTime,
+};
```

Comment:
Is it a good idea to have this object and not two separate exports. I'm thinking of tree shaking, where all helpers would be imported, regardless if they're used or not. I know, we don't care about bundle size in the Admin at the moment, but I guess that will change once we optimize for mobile.

Reply:
I just did this so the exported values include `gridColumnTypes` because exporting values like `date` and `dateTime` were too generic.

Should we export the two variables like this, instead of the object? 

- `dateGridColumn`
- `dateTimeGridColumn`

Or:

- `dateGridColumnType`
- `dateTimeGridColumnType`

Reply:
Changed in d6eb9b9c9b557f659c39f5930bd78a587825806d